src/pages/Add.js

```diff
@@ -42,27 +44,21 @@ export default function Add({ handleInputChange }) {
     <>
       <GlobalStyles />
 
-      <>
-        <StyledPageDiv>
-          <Seachbar
-            handleInputChange={setSearch}
-            onSearch={handleSearch}
-            onChange={event => handleInputChange(event.target.value)}
-          />
+      <StyledPageDiv>
+        <Seachbar
```

Comment:
I think you meant "Searchbar"

Reply:
<img width="336" alt="Screenshot 2019-10-17 at 16 16 31" src="https://user-images.githubusercontent.com/54042879/67017109-8c20e600-f0f9-11e9-9148-c4f422551c20.png">